.env

```diff
@@ -1 +1 @@
-FLASK_ENV=production
\ No newline at end of file
+FLASK_ENV=development
```

Comment:
It'd probably be best to leave this on `production` mode. `development` mode seems to impact performance.

Reply:
You're right I did not intend to push that change, I prefer the development while running flask locally as it reloads on save.